### DISCUSSION TITLE
Unable to take a clear picture of proof of education status because the form only enables the front camera

### DISCUSSION THREAD

**Discussion by afuhflynn** [community/community#146111] [GitHub Education]
### Hi everyone,

 When I try taking a picture of my document for proof of educational status the back camera doesn't function hindering me from taking a clear picture and the toggle for back camera doesn't work. I will love if someone can help with this.

**Comment by quoc10973** [comment]
You use laptop or phone to take photo?, i suggest using phone for better ID card quality photo

**Reply by afuhflynn** [reply]
In used my phone but the back camera is not enabled when I click on the take photo button. Only the front camera is active.

**Reply by quoc10973** [reply]
well, try take photo that incldue your face and your hand is showing ID card. I did successfully

**Reply by afuhflynn** [reply]
Okay 👍 thank you for your time
